front/style/pages/_register.scss

```diff
@@ -89,6 +89,22 @@
         &--flex {
             display: flex;
             align-items: center;
+
+            #form__mandate {
+                .select2-container {
+                    width: 100%;
+
+                    .select2-search-choice-close {
+                        border-bottom: unset;
+                    }
+                }
+
+                .form__field {
+                    padding-left: unset;
```

Comment:
Same here, and also for the `padding-right` and `height`

Reply:
:heavy_check_mark: